lutron-aurora/src/init.lua

```diff
@@ -0,0 +1,127 @@
+local zcl_clusters = require "st.zigbee.zcl.clusters"
+local Level = zcl_clusters.Level
+local ZigbeeDriver = require "st.zigbee"
+local capabilities = require "st.capabilities"
+local device_management = require "st.zigbee.device_management"
+local utils = require "st.utils"
+local zcl_commands = require "st.zigbee.zcl.global_commands"
+local read_responder = require "read_responder"
+
+local PowerConfiguration = zcl_clusters.PowerConfiguration
+local Groups = zcl_clusters.Groups
+
+local LAST_LEVEL_EVENT = "LAST_LEVEL_EVENT"
+local DEVICE_GROUP = "DEVICE_GROUP"
+--server: Basic, PowerConfiguration, Identify, TouchlinkCommissioning, 0xFC00 
+--client: Identify, Groups, OnOff, Level, OTAUpgrade, TouchlinkCommissioning
+
+local function do_configure(self, device)
+  if not self.environment_info.hub_zigbee_eui then
+    return self:call_with_delay(1, function()
+      do_configure(self, device)
+    end)
+  end
```

Comment:
were you running into this happening during testing?

Reply:
Not really, just when I was trying to get some already onboarded devices reconfigured so I was calling this from `added`